styles.scss

```diff
@@ -712,6 +712,10 @@ q {
     }
   }
 }
+.ElementDelete {
+  margin-left: auto;
+  width: 31px;
+}
```

Comment:
Any reason this one isn't wrapped within `ElementFormWrapper` like the other element styling rules?

Reply:
The `ElementFormWrapper` is for the management of base elements (food ingredients and textile materials), but the `ElementDelete` for the delete button is used throughout the simulators, not only in the "base elements steps"